models/pattern/core/pattern.go

```diff
@@ -370,7 +370,9 @@ func createPatternServiceFromCoreK8s(manifest map[string]interface{}) (string, S
 	namespace, _ := metadata["namespace"].(string)
 	labels, _ := metadata["labels"].(map[string]interface{})
 	annotations, _ := metadata["annotations"].(map[string]interface{})
-
+	if namespace == "" {
```

Comment:
Let's confirm that objects that are not namespaced are not affected by this logic; that non-namespaced objects are NULL, not empty string.

Reply:
Do we have a null vs empty string concern?